elements/pfe-navigation/demo/single-nav.html

```diff
@@ -43,7 +43,7 @@
     ])</script>
   </head>
   <body unresolved style="background-color: #ddd;">
-     <pfe-navigation id="pfe-navigation-2">
+    <pfe-navigation id="pfe-navigation-2">
       <nav class="pfe-navigation" aria-label="Main Navigation">
         <div class="pfe-navigation__logo-wrapper" id="pfe-navigation__logo-wrapper">
```

Comment:
I strongly advise against requiring classes in light DOM in order to upgrade the component.  This is a poor author experience because if you make a typo or put them in the wrong order, the entire thing breaks and our team will get a lot of calls to "fix" the component when it's a case of poor implementation.  My approach has been build the simplest light DOM mark-up possible, focus on the required tags for graceful degradation, and then build the component up from there.  Slots can help with moving items into their appropriate locations so that if the user puts them in the wrong place, you can grab and move it easily.  Happy to break out on this.

Reply:
@castastrophe good point, @wesruv wanted to make sure you saw the above comment. I do think this is a larger conversation outside of this PR, so maybe we should all regroup on this.

Reply:
Yea, we can talk about this in our upcoming meeting.